comparisons/validateJR.sh

```diff
@@ -27,14 +27,13 @@ grep root ${inList} | grep -v "#" | while read -r dsN fNP procN comm; do
     #[ ! -f "${baseA}/${fN}" ] && echo Missing ${baseA}/${fN}
     #process regular files first; need files both in baseA and baseB
     if [ -f "${baseA}/${fN}" -a -f "${baseB}/${fN}" ]; then
-	extN=all_${diffN}_${dsN}
-	mkdir -p ${extN}
-	pushd ${extN}
-	  cp ${VALIDATE_C_SCRIPT} ./
-	  echo "Will run on ${fN} in ${extN}"
-	  g++ -shared -o validate.so validate.C `root-config --cflags ` -fPIC
-	  echo -e "gSystem->Load(\"libFWCoreFWLite.so\");\n AutoLibraryLoader::enable();\n FWLiteEnabler::enable();\n
-                  .x validate.C+(\"${extN}\", \"${baseA}/${fN}\", \"${baseB}/${fN}\", \"${procN}\");\n .qqqqqq" | root -l -b >& ${extN}.log &
+        extN=all_${diffN}_${dsN}
```

Comment:
@vlimant , looks like the `validate.so` we build is not used and `root -l -b` generates `validate_C.so` on fly. So is there any reason to keep `g++ -shared -o validate.so validate.C `root-config --cflags ` -fPIC` ?